sync-mixins.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TMPDIR="./tmp"
+MIXIN_REPO="git@github.com:giantswarm/giantswarm-kubernetes-mixin.git"
+# clone a branch if provided
+BRANCH=${1:-""}
+
+# make a temporary dir to work in
+mkdir ${TMPDIR}
+
+if [ -z "${BRANCH}" ]; then
+    # clone the mixins repo
+    echo -e "\nCloning master branch:\n"
+    git clone ${MIXIN_REPO} ${TMPDIR}/mixins
+else
+    # clone the mixins repo branch
+    echo -e "\nCloning branch '${BRANCH}':\n"
+    git clone -b ${BRANCH} --single-branch ${MIXIN_REPO} ${TMPDIR}/mixins
```

Comment:
Let's use long arg name in script to make things more readable.

```suggestion
    git clone --branch ${BRANCH} --single-branch ${MIXIN_REPO} ${TMPDIR}/mixins
```